frontend/packages/dev-console/src/components/pipelines/__tests__/PipelinesPage.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineModel } from '../../../models';
+import { PipelinesPage } from '../PipelinesPage';
+import ProjectListPage from '../../projects/ProjectListPage';
+import PipelinesResourceList from '../PipelinesResourceList';
+
+describe('Pipeline List', () => {
+  it('Should render a PipelineResourcelist', () => {
+    const pipelinePageProps: React.ComponentProps<typeof PipelinesPage> = {
+      history: null,
+      location: null,
+      match: {
+        isExact: true,
+        path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+        url: 'k8s/ns/my-project/tekton.dev~v1alpha1~Pipeline',
+        params: {
+          ns: 'my-project',
+        },
+      },
+    };
+    const pipelineWrapperNS = shallow(<PipelinesPage {...pipelinePageProps} />);
+    expect(pipelineWrapperNS.find(PipelinesResourceList).exists()).toBe(true);
+  });
+
+  it('Should render ProjecListPage when no namespace is selected', () => {
+    const pipelinePagePropsWNS: React.ComponentProps<typeof PipelinesPage> = {
+      history: null,
+      location: null,
+      match: {
+        isExact: true,
+        path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+        url: 'k8s/ns/all-projects/tekton.dev~v1alpha1~Pipeline',
+        params: {
+          ns: undefined,
+        },
+      },
+    };
+    const pipelineWrapperWNS = shallow(<PipelinesPage {...pipelinePagePropsWNS} />);
+    expect(pipelineWrapperWNS.find(PipelinesResourceList).exists()).toBe(false);
```

Comment:
Why do you need this check here? Is there any use-case when both components are going to be on the page? I dont think so at any given time both lists will be shown on the page. Also if you want to add this check here then in the above test (when a namespace is selected) you should have this check : 
```
 expect(pipelineWrapperWNS.find(ProjectListPage).exists()).toBe(false);
```

Reply:
It is exactly like you said. It is not a use case but a mandate that for namespace = undefined, ProjectList must be true and ResourceList must not be true. I see nothing wrong here.

Reply:
Yeah nothing wrong in adding that check but its redundant, and if you are adding check here why not in (when a namespace is selected). Look at the above comment.

Reply:
removed